config/routes.rb

```diff
@@ -405,6 +405,7 @@
     resources :people, :programmes, :projects, :assays, :studies, :models, :sops, :workflows, :data_files, :publications, :documents, only: [:index]
     member do
       get :export_isatab_json
+      get :export_isa, action: :export_isa
```

Comment:
You should delete the route in the single_pages resource since it's not used anymore:
https://github.com/seek4science/seek/blob/bf3200c8d4c8c9b921269b5b233a76299394dc79/config/routes.rb#L754

Reply:
Good catch - fixed in [60fcdde](https://github.com/seek4science/seek/pull/1698/commits/60fcddee7fdfdeac3ece58fc81c8df8b8798b260)